app/models/medium.rb

```diff
@@ -580,6 +580,14 @@ def sort_localized
     Medium.sort_localized[sort]
   end
 
+  def subheader_style
```

Comment:
Similar as above. It would be more ruby-esque to use a guard clause. The `self` is redundant.
```
def subheader_style
  return "badge badge-secondary" unless sort == "nuesse" && file_last_edited
  "badge badge-danger"
end
```